rasa/shared/nlu/training_data/message.py

```diff
@@ -29,6 +30,14 @@
 
 
 class Message:
+    """Container for data that describes one conversation state.
+
+    The data describes a set of attributes that can also be used to describe
+    at most one use and one action sub-state. For example, these attributes
```

Comment:
I think this is a typo
```suggestion
    at most one user and one action sub-state. For example, these attributes
```

Reply:
let me double-check with @ka-bu - in your Message docstring did you mean `use` or `user`?

Reply:
sorry that was a typo, I meant "user" :) a state has sub-states describing e.g. a user utterance or a bot action